pom.xml

```diff
@@ -40,9 +103,9 @@
 			<version>3.3.1-SNAPSHOT</version>
 		</dependency>
 		<dependency>
-			<groupId>com.sk89q</groupId>
-			<artifactId>worldedit</artifactId>
-			<version>6.0.0-SNAPSHOT</version>
+			<groupId>com.sk89q.worldedit</groupId>
+			<artifactId>worldedit-bukkit</artifactId>
```

Comment:
As soon as I can make a few changes to sk_pos1 and 2, there will be no references to Bukkit at all in the WorldEdit class.